contribs/gmf/src/controllers/abstractdesktop.js

```diff
@@ -142,6 +144,21 @@ gmf.AbstractDesktopController = function(config, $scope, $injector) {
    */
   this.profileLine = null;
 
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.drawProfilePanelActive = false;
+
+  /**
+   * The ngeo ToolActivate manager service.
+   * @type {ngeo.ToolActivateMgr}
+   */
+  var ngeoToolActivateMgr = $injector.get('ngeoToolActivateMgr');
```

Comment:
Even if the tool is not present in the mobile app, all this code should be moved to `contribs/gmf/src/controllers/abstract.js`